repo/fsrepo/fsrepo.go

```diff
@@ -482,6 +482,14 @@ func (r *FSRepo) SetConfigKey(key string, value interface{}) error {
 		return err
 	}
 
+	// Load private key to guard against it being overwritten.
+	// NOTE: this is a temporary measure to secure this field until we move
+	// keys out of the config file.
+	pkval, err := common.MapGetKV(mapconf, "Identity.PrivKey")
```

Comment:
@jbenet as we will be moving the key out of config in the nearest future, can we accept current hack (it is hack) and release keymigrations in 0.4.4 or 0.5.

This isn't long term strategy, it is a way to keep things running for till next release.